app/src/main/java/com/example/graburger/MainActivity.java

```diff
@@ -160,13 +150,40 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
             BurgerItemModel orderedBurger = (BurgerItemModel) data.getExtras().getSerializable("order");
             orderedBurger.updateBurger();
             if (requestCode == buildBurgerActivity.CREATE_NEW_ACTIVITY_CODE) {
-                orderList.add(orderedBurger);
+                orderList.add(0, orderedBurger);
```

Comment:
Why are you adding the burger in position 0?

Reply:
So the last added item to the order will be shown to the user when added